tools/integration_tests/readonly/create_object_test.go

```diff
@@ -37,11 +37,13 @@ func ensureFileSystemLockedForFileCreation(filePath string, t *testing.T) {
 
 func TestCreateFile(t *testing.T) {
 	filePath := path.Join(setup.MntDir(), "testFile.txt")
+
 	ensureFileSystemLockedForFileCreation(filePath, t)
```

Comment:
you are checking if the file creation failed or not, why is your method name ensureFileSystemLocked ? can you rename in a simpler way - checkIfFileCreationFailed .

Same for all other methods.

Reply:
@raj-prince suggested this name.